hm-post-repeat.php

```diff
@@ -155,9 +159,16 @@ function save_post_repeating_status( $post_id = null, $post_repeat_setting = nul
 
 	if ( 'no' === $post_repeat_setting ) {
 		delete_post_meta( $post_id, 'hm-post-repeat' );
+		return;
 	}
 
-	else {
+	// Backwards compatibility.
```

Comment:
I've removed the extra check and updated the tests to use the new schedule format.